src/.vuepress/theme/styles/layout/_tabs.scss

```diff
@@ -44,6 +44,7 @@
     padding-left: 14px;
     margin: 0 0 -5px 0;
     opacity: 0.5;
+    text-transform: capitalize;
```

Comment:
We should do the same for the mobile view (See "use" and "extend"): 
![image](https://user-images.githubusercontent.com/44427849/61617281-aa965300-ac6a-11e9-9f5b-157cabec9167.png)